mininext/node.py

```diff
@@ -92,17 +96,25 @@ def startShell(self):
         self.lastCmd = None
         self.lastPid = None
         self.readbuf = ''
-        self.waiting = False
-
         # If this node has a private PID space, grab the PID to attach to
         # Otherwise, we use the same PID as the shell's PID
         if self.inPIDNamespace:
             # monitor() will grab shell's true PID and put in self.lastPid
-            self.monitor()
+            self.monitor(1000)
```

Comment:
Reasonable to have a timeout -- not sure if Mininet had this support before, but certainly better than hanging.